kafka-connect/build.gradle

```diff
@@ -64,9 +64,14 @@ project(':iceberg-kafka-connect:iceberg-kafka-connect-runtime') {
   configurations {
     hive {
       extendsFrom runtimeClasspath
+      // force upgrades for dependencies with known vulnerabilities...
+      resolutionStrategy {
+        force 'commons-beanutils:commons-beanutils:1.11.0'
```

Comment:
I think we keep it under the `resolutionStrategy`  at line 76?

Reply:
My thinking was that we didn't want/need this for the "all" target because that's a Hive dependency. That said, moving it here didn't include it in the non-hive distZip build so I've done as you asked.